settings.py

```diff
@@ -390,7 +390,8 @@
 
     'custom.icds_reports',
     'custom.enikshay.integrations.ninetyninedots',
-    'custom.pnlppgi'
+    'custom.pnlppgi',
+    'custom.enikshay.nikshay_datamigration',
```

Comment:
nit, but can we keep all enikshay custom apps together? Since we are doing this more often, might even be nice  to have an `ENIKSHAY_APPS` and do `INSTALLED_APPS + ENIKSHAY_APPS` or something.